test/OpenTelemetry.Tests/Impl/Trace/TracingTest.cs

```diff
@@ -62,17 +62,17 @@ public void DefaultTracer()
             Assert.Same(Tracer.NoopTracer, Tracing.Tracer);
         }
 
-        [Fact(Skip = "need to fix the way tracer being instantiated")]
-        public void DefaultTraceExporter()
-        {
-            Assert.Equal(ExportComponent.NewNoopExportComponent.GetType(), Tracing.ExportComponent.GetType());
-        }
+        //[Fact(Skip = "need to fix the way tracer being instantiated")]
+        //public void DefaultTraceExporter()
+        //{
+        //    Assert.Equal(ExportComponent.NewNoopExportComponent.GetType(), Tracing.ExportComponent.GetType());
```

Comment:
is it intentionally commented? should it be returned back or fixed?

Reply:
Updated to restore tests